aptos-move/aptos-vm-types/src/change_set.rs

```diff
@@ -40,47 +40,48 @@ pub struct GroupWrite {
     /// derive metadata_op will be validated during parallel execution to make sure
     /// it is correct, and the bytes will be replaced after the transaction is committed
     /// with correct serialized group update to obtain storage WriteOp.
-    metadata_op: WriteOp,
+    pub metadata_op: WriteOp,
     /// Updates to individual group members. WriteOps are 'legacy', i.e. no metadata.
     /// If the metadata_op is a deletion, all (correct) inner_ops should be deletions,
     /// and if metadata_op is a creation, then there may not be a creation inner op.
     /// Not vice versa, e.g. for deleted inner ops, other untouched resources may still
     /// exist in the group. Note: During parallel block execution, due to speculative
     /// reads, this invariant may be violated (and lead to speculation error if observed)
     /// but guaranteed to fail validation and lead to correct re-execution in that case.
-    inner_ops: BTreeMap<StructTag, (WriteOp, Option<Arc<MoveTypeLayout>>)>,
+    pub inner_ops: BTreeMap<StructTag, (WriteOp, Option<Arc<MoveTypeLayout>>)>,
```

Comment:
I think we can keep inner_ops private, but instead make `inner_ops()` return an iterator?